src/Pages/Login.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useRef, useContext } from 'react'
+import { AuthContext } from '../store/auth-context';
+import { supabase } from '../supabase/supabase-client'
+
+const Login = () => {
+  const emailInputRef = useRef();
+  const passwordInputRef = useRef();
+
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  const authCtx = useContext(AuthContext);
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+    setLoading(true);
+    const { user, session, error } = await supabase.auth.signIn({
+      email: emailInputRef.current.value,
+      password: passwordInputRef.current.value,
```

Comment:
Vidím, že tady používáš "uncontrolled" inputy. To je v pohodě, dokud rozumíš rozdílu mezi "controlled" a "uncontrolled" inputy. Já bych při review uncontrolled inputy určitě vždycky komentoval, protože mi to nepřijde, jako ten "správny React způsob", ale není to nic proti ničemu.